### DISCUSSION TITLE
关于SLANet_lcnetv2

### DISCUSSION THREAD

**Discussion by wu-yz** [PaddlePaddle/PaddleOCR#13816] [Q&A]
请问SLANet_lcnetv2的推理模型是不是还用不了？我使用predict_table.py进行表格识别，将模型换成SLANet_lcnetv2却效果不好，请问这是什么原因呀？

**Comment by GreatV** [comment]
给的信息太少了，按照说明文档来应该没有问题的。
https://paddlepaddle.github.io/PaddleOCR/algorithm/table_recognition/algorithm_table_slanet.html#4

**Comment by UserWangZz** [comment]
目前关于表格识别确实是一个很hard的任务，如果最新模型不及预期，可以尝试重新微调模型

**Comment by lllzqq** [comment]
SLANet_lcnetv2的推理模型复现效果非常不好啊 
执行以下命令测试推理模型效果特别差
python table/predict_structure.py --table_model_dir=../inference/slanet_lcnetv2_infer/ --table_char_dict_path=../ppocr/utils/dict/table_structure_dict.txt --image_dir=docs/table/1.png --output=../output/table_slanet_lcnetv2 --use_gpu=False --benchmark=False --enable_mkldnn=True --table_max_len=512
